CHANGELOG.md

```diff
@@ -1,11 +1,179 @@
+- Project developers default to having access permissions for all libraries within the project, consistent with previous version behavior
+- New project participant role added, participants default to no access permissions for any library
+- Participants can apply for library access permissions through tickets
+- Administrators can directly authorize participants with library access permissions
+- Administrators can revoke library access permissions from participants
+
+Notification Messages
+
+- Project collaboration adds message notification feature
+- Supported event types include work order approval status change, task execution status change, and task scheduling failures
+- Notification range can be configured through rules, message content can be configured through templates
+- Notification channels support configuring common webhook channels such as DingTalk, Lark, WeCom, also support custom HTTP requests, message sending supports rate limiting
+
+System Integration
+
+- SSO adds LDAP protocol support
+
+### Usability Improvements
+
+- Optimized the database selection component, standardizing the library selection interaction across product pages and adding fuzzy matching functionality for project names, data source names, and database names.
+- Added a resource tree locator key to the SQL window for quickly identifying the current database's position within the resource tree.
+- Upgraded preference settings to a top-level feature accessible directly through the [Settings] entry; new configuration options include:
+    - Whether to enable full-link tracing during SQL execution.
+    - Whether to continue SQL execution upon encountering errors.
+    - Customization of editor theme and font size.
+    - Configuration of editor shortcuts for running SQL commands.
+    - Setting the default workspace.
+    - Whether to enable user behavior analytics.
+    - Desktop version now supports memory size configuration through JVM parameters.
+- Added database availability indicators; the database list under projects will now show unavailable statuses and reasons.
+- Improved the initiation interaction for work orders:
+    - Support for initiating various work orders directly from the database in the resource tree.
+    - Common task work orders (mock data, database changes, data archiving, data cleanup, database permission application) support reinitiation with editable task parameters.
+- Enhanced database change processes to detect index change statements and automatically adjust the timeout setting (120h) to prevent index change statement execution failure due to timeout.
+- Desktop version personal settings now support custom JVM configuration with memory usage control to within 1 GB.
+- Desktop version supports exporting data files larger than 4 GB.
+- Optimized the en-US language wording of the product.
+
+### Breaking Changes
+
+- The default environment for security standards no longer supports editing or disabling.
```

Comment:
The default environment for security standards continues to supports editing or disabling.

Reply:
removed